package.json

```diff
@@ -87,6 +87,7 @@
     "qrcode": "^1.4.4",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
+    "react-feather": "^2.0.3",
```

Comment:
🤔 (Encourage to use icons in `@material-ui/icons` to create a more consistent vision...)